### PR TITLE
Add signatures.json

### PR DESCRIPTION
**Pull request by robhudson**

### CODE REVIEW

File: basket/petition/views.py [hunk 1]
```diff
@@ -3,8 +3,10 @@
 from django.conf import settings
 from django.http import JsonResponse
 from django.shortcuts import redirect
+from django.views.decorators.cache import cache_page
```
Comment (stevejalim) [inline basket/petition/views.py:6]: I'm assuming that the basket prod setup has a real redis/memcached cache configured, yep?

By defualt, we're using locmem

https://github.com/mozmeao/basket/blob/6d8aa5582420906b6b37590056114a0c146894b0/basket/settings.py#L73-L84

**Review by stevejalim — APPROVED** [review]
🎸 r+wc

File: basket/petition/views.py (hunk 1 above)
Comment (pmac) [reply basket/petition/views.py:6]: I believe we do. We can check with SRE. Either way this seems like a good thing.

**Review by pmac — APPROVED** [review]
🖊️

File: basket/petition/views.py (hunk 1 above)
Comment (robhudson) [reply basket/petition/views.py:6]: `CACHE_URL` gets set if we have a `REDIS_URL`, so this should go into the Redis cache. Just above this block of code.

**Pull request merged**